src/util/http.ts

```diff
@@ -58,3 +59,11 @@ export const createErrorRes = ({
     }),
   };
 };
+
+export const decodeToken = ((token : any) => {
+  try {
+    return jwt.verify(token, process.env.JWT_SECRET);
+  } catch (e) {
+    return null;
```

Comment:
```suggestion
    console.error(e);
    return null;
```
에러가 발생했을 시, 에러를 로그에 남겨야, 추후 서비스 운영시에 에러가 났을때 대응하기 수월해집니다.